### DISCUSSION TITLE
Embedded Image

### DISCUSSION THREAD

**Discussion by hoellersbuero** [haplokuon/netDxf#353] [Q&A] [answered]
I am creating DXF files with my software. I also include png files using the image class, but by this way they are referenced, not embedded.
Is there a possibility to embed the picure in the DXF so that it is seen on another device?

Regards, Michael

**Comment by haplokuon** [comment, marked as answer]
The list of supported entities is in the README.md, the information of not supported entities is lost. Embedded images are stored as OLE or OLE2 entities and they are not among them.
